assets/_scss/elements/_inputs.scss

```diff
@@ -78,6 +81,17 @@ select {
   background-position: right 1.3rem center;
 }
 
+legend {
```

Comment:
@msecret I wrote a different rule, to resolve this issue here: 5d94448. Basically adding this alternate font size directly to the error message class.